run_F2010SC5-CMIP6_E3SMv2_alpha5_59_wP3_cori.csh

```diff
@@ -0,0 +1,1622 @@
+ clubb_c6thlc           = 0.50       !! same as in alpha22
+ clubb_c8               = 5.2        !! same as in alpha22
+ clubb_c_k10            = 0.35       !! same as in alpha22
+ clubb_c_k10h           = 0.35       !! should be the same as clubb_c_k10
+ clubb_gamma_coef       = 0.12D0     !! same as in alpha22
+ clubb_gamma_coefb      = 0.28D0     !! same as in alpha22
+ clubb_gamma_coefc      = 1.2        !! same as in alpha22
+ clubb_mu               = 0.0005     !! same as in alpha22
+ clubb_wpxp_l_thresh    = 100.0D0    !! same as in alpha22
+ clubb_ice_deep         = 14.e-6     !! same as in alpha22
+ clubb_ice_sh           = 50.e-6     !! same as in default
+ clubb_liq_deep         = 8.e-6      !! same as in default
+ clubb_liq_sh           = 10.e-6     !! same as in default
+ clubb_C2rt             = 1.75D0     !! same as in default
+ clubb_use_sgv          = .true.
+ seasalt_emis_scale     = 0.6
+
+!
+!- zm
+!
+ zmconv_c0_lnd          = 0.0020     !! 0.007 in default
+ zmconv_c0_ocn          = 0.0020     !! 0.007 in default
+! zmconv_dmpdz           =-1.2e-3     !! -0.7e-3 in default
+ zmconv_ke              = 5.0E-6     !! same as in default
+ zmconv_alfa            = 0.14D0
+ zmconv_tp_fac          = 2.0D0
+ zmconv_tiedke_add      = 0.8D0      !! same as in default
+ zmconv_cape_cin        = 1          !! same as in default
+ zmconv_mx_bot_lyr_adj  = 1          !! 2 in default
+
+!
+!- microphysics
+!
+ prc_coef1               = 30500.0D0 !! same as in default
+ prc_exp                 = 3.19D0    !! same as in default
+ prc_exp1                = -1.40D0   !! -1.2D0  in default
+ micro_mg_accre_enhan_fac = 1.75D0   !! 1.5D0   in default
+ microp_aero_wsubmin     = 0.001D0   !! was missing
+ so4_sz_thresh_icenuc    = 0.080e-6  !! 0.05e-6 in default
+ relvar_fix              = .true.    !! same as in default
+ mg_prc_coeff_fix        = .true.    !! same as in default
+ rrtmg_temp_fix          = .true.    !! same as in default
+ micro_mg_berg_eff_factor = 0.7D0    !! same in v1p and alpha22
+! ice_sed_ai              = 1200.0    !! 500.0   in default
+! cld_sed                 = 1.8D0
+ cldfrc_dp1              = 0.018D0   !! alpha22 0.1D0
+! effgw_oro               = 0.25      !! same as in default
+! effgw_beres             = 0.4       !! same as in default
+ do_tms                  = .false.   !! same as in default
+ cldfrc2m_rhmaxi         = 1.05D0    !! same as in default
+ n_so4_monolayers_pcage  = 8.0D0     !! same as in default
+ taubgnd                 = 2.5D-3    !! same as in default
+ raytau0                 = 5.0D0     !! same as in default 
+
+! state_debug_checks = .true.   ! turn on if wanting to debug crashes
+ history_budget = .true.
+ nucleate_ice_subgrid = 1.2d0
+ use_hetfrz_classnuc = .true.
+ do_Cooper_inP3 = .false.
+
+EOF
+
+cat >> user_nl_elm <<EOF
+
+ check_finidat_year_consistency = .true.
+ finidat = '/compyfs/linw288/inputdata/20201103.IELM.r05_oECv3.elm.r.0030-01-01-00000.nc'
```

Comment:
@crterai 
Here I guess you want to use the file on Cori (please check that it's accessible for reading):
`/global/cfs/cdirs/e3sm/shpundk/elm_inputdata/20201103.IELM.r05_oECv3.elm.r.0030-01-01-00000.nc`

Reply:
I was able to read this so. It has been changed